### ISSUE TITLE
Hello, does the project still exist?

### ISSUE THREAD

**Issue by Sascha321** [butor/blackbird#518]
Hello, does the project still exist?

**Comment by mongrelion** [comment]
Yes, it does. The last change to the code was on the 26th of August, 2 days before your question.

**Comment by yllierop** [comment]
I'm working on this project at: https://github.com/yllierop/blackbird and trying to make it work in the modern BTC world.

**Comment by mxcl** [comment]
> The last change to the code was on the 26th of August

That was an update to the README, the code hasn’t been modified in 3 years. Project is dead, though possibly is still functional enough—I have not tried it out.
